contentcuration/contentcuration/frontend/shared/views/StudioChip.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+
+  <div
+    ref="chip"
+    class="studio-chip"
+    :style="chipStyles"
+  >
+    <div class="studio-chip__content">
+      <div class="studio-chip__text">
+        <slot>
+          {{ text }}
+        </slot>
+      </div>
+
+      <KIconButton
+        v-if="close"
+        class="studio-chip__close"
+        :aria-label="removeLabel"
+        icon="delete"
+        :color="$themePalette.grey.v_400"
+        data-test="remove-chip"
+        @click.stop="handleClose"
+      />
+    </div>
+  </div>
+
+</template>
+
+
+<script>
+
+  export default {
+    name: 'StudioChip',
+    props: {
+      text: {
+        type: String,
+        default: '',
+      },
+      close: {
+        type: Boolean,
+        default: false,
+      },
+    },
+    computed: {
+      chipStyles() {
+        const baseColor = this.$themePalette.grey.v_200;
+        return {
+          backgroundColor: baseColor,
+        };
+      },
+      removeLabel() {
+        return `Remove ${this.text}`;
+      },
+    },
+    methods: {
+      handleClose() {
+        this.$emit('close');
+      },
+    },
+  };
+
+</script>
+
+
+<style lang="scss" scoped>
+
+  .studio-chip {
+    display: inline-flex;
+    align-items: center;
+    height: 26px;
+    min-height: 26px;
+    padding: 2px 8px;
+    margin: 2px;
+    font-size: 13px;
+    white-space: nowrap;
+    cursor: default;
+    user-select: none;
+    border-radius: 12px;
+    transition: all 0.2s ease;
+
+    &__content {
```

Comment:
Please see [here](https://github.com/learningequality/studio/pull/5345#discussion_r2333230796) on BEM - not only for this PR, but also other ones.